include/ensmallen_bits/problems/ackley_function.hpp

```diff
@@ -47,7 +47,8 @@ class AckleyFunction
    * @param c Multiplicative constant with a default value of 2 * pi.
    * @param epsilon Coefficient to avoid division by zero (numerical stability).
    */
-  AckleyFunction(const double c = 2 * arma::datum::pi,
+  AckleyFunction(const double n = 2,
```

Comment:
Can you add a comment for the number of dimensions. Also, I think this isn't part of the public API, so we don't break compatibility.

Reply:
Agreed, the test functions aren't documented anywhere, so for now I wouldn't consider them public.  It might be nice to make them public at some point, but I'm not sure how useful people would find them, and I don't think we need to do that now.